android/gradle.properties

```diff
@@ -11,8 +11,8 @@
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
 #Thu Mar 03 15:15:24 EST 2022
```

Comment:
is this file supposed to be generated by a script 🤔

Reply:
Nope, it's hand-crafted (unless some RN tooling generates it?). You can probably delete this line